plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/utils/HuaweiCloudToXpanseDataModelConverter.java

```diff
@@ -164,10 +164,10 @@ public List<Metric> convertBatchListMetricDataResponseToMetric(
                                 datapointForBatchMetrics.sort(
```

Comment:
The returned data is inherently ordered, and there is no need to sort the list again.

Reply:
Okay, I have made the changes.